model/src/w3gdatmd.F90

```diff
@@ -2299,12 +2271,10 @@ SUBROUTINE W3SETG ( IMOD, NDSE, NDST )
     US3DF  => GRIDS(IMOD)%US3DF
     USSPF  => GRIDS(IMOD)%USSPF
     USSP_WN => GRIDS(IMOD)%USSP_WN
+    FFACBERG => GRIDS(IMOD)%FFACBERG
```

Comment:
@MathieuDutSik this line looks added?  Is this intentional? Is this moving something from somewhere else (couldn't find where that was if so, but might have just missed it).

Reply:
@MathieuDutSik , okay that checks out.  Thanks for the clarification.